test/unit/org/scratchOrgInfoApi.test.ts

```diff
@@ -466,29 +466,48 @@ describe('authorizeScratchOrg', () => {
     sandbox.restore();
   });
 
-  it('authorizeScratchOrg', async () => {
+  it('authorizeScratchOrg basic', async () => {
     hubOrgStub.isDevHubOrg.returns(true);
     const result = await authorizeScratchOrg({
       scratchOrgInfoComplete: TEMPLATE_SCRATCH_ORG_INFO,
       hubOrg: hubOrgStub,
     });
 
     expect(result).to.be.equal(authInfo);
+    expect(authInfoStub.firstCall.args[0]).to.deep.equal({
+      username: TEMPLATE_SCRATCH_ORG_INFO.SignupUsername,
+      parentUsername: username,
+      oauth2Options: {
+        loginUrl: `https://${TEMPLATE_SCRATCH_ORG_INFO.SignupInstance}.salesforce.com`,
+        privateKeyFile: privateKey,
+      },
+    });
```

Comment:
None of the existing tests could possibly fail since `result` is a stubbed return setup by the tests.  I added expectations that should have been there from the start, as well as a few additional checks for the new code I added.